src/core/config/remote/remote_config_data_wrapper.ts

```diff
@@ -39,13 +33,15 @@ export class RemoteConfigDataWrapper implements Validate, ToObject<RemoteConfigD
   private _components: ComponentsDataWrapper;
   private _commandHistory: string[];
   private _lastExecutedCommand: string;
+  private readonly _flags: CommonFlagsDataWrapper;
```

Comment:
```suggestion
  private readonly flags: CommonFlagsDataWrapper;
```
https://google.github.io/styleguide/tsguide.html#naming-style : Do not use trailing or leading underscores for private properties or methods.

Reply:
With Nathan we discussed and agreed that values that have a getter `get flags()` we will use _underscore prefix in naming convention, this is same as for the hedera javascript sdk